README.md

```diff
@@ -16,11 +16,12 @@ Once you've fetched the scripts, you can update them with `git pull`
 
 The target platform is Microsoft Windows.
 
-## Organization
+## File Organization
 
-We don't have much software defined yet.  So the organization is probably wrong.
-
-* capture -- for scripts to assist in capturing images (e.g. SharpCap)
-* analysis -- image processing scripts
+We've set this up to produce a python 'package' called 'debi' which should
```

Comment:
The package name 'debi' is not available in pypi, so that should be changed to something we can control.